inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -120,9 +120,11 @@ def from_random_depths(
             random.uniform(-max_depth, 0),
         ]
 
-        _, _, depths = inductiva.generative.procedural.generate_random_terrain(
-            x_range, y_range, x_num, y_num, corner_values, initial_roughness,
-            roughness_factor, percentile_above_water)
+        depths = inductiva.generative.procedural.generate_random_terrain(
```

Comment:
Why don't you go more specific and call this:

```
inductiva.generative.procedural.generate_random_depth_map(
```

A terrain has physical properties. A depth map is an abstract thing on a grid.